src/main/webapp/content/scss/global.scss

```diff
@@ -413,3 +413,27 @@ global styles
   background: #f6f6f6;
   box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
 }
+
+.panel-content-wrapper {
```

Comment:
Maybe add a `cursor: pointer;` to the wrapper so that the cursor looks like a pointer on hover (not sure if you need to add a `.panel-content-wrapper:hover`, please check)